syntax = "proto3";

package Packet;

message FakeCluePacket {
	uint32 time = 1;
	uint32 player_id = 2;
}